drivers/google/google.go

```diff
@@ -242,7 +243,11 @@ func (d *Driver) Start() error {
 	if err != nil {
 		return err
 	}
-	return c.createInstance(d)
+	if err = c.createInstance(d); err != nil {
+		return err
+	}
+	d.IPAddress, err = d.GetIP()
```

Comment:
I think I originally meant to `return err` below, but then I thought it would be weird to return GetIP's error. But that's probably not at all weird, so I'll fix this :)

(this also applies to all the places I did `d.IPAddress, _ = d.GetIP()`)